python/paddle/hapi/model_summary.py

```diff
@@ -35,6 +35,7 @@ def summary(net, input_size, dtypes=None):
                     have multiple input, input_size must be a list which contain 
                     every input's shape. Note that input_size only dim of
                     batch_size can be None or -1.
+        input (dict|list|): the input tensor. 
```

Comment:
input can be of any type. And need to explain, if given, input_size and dtype will be ignored.

Reply:
ok